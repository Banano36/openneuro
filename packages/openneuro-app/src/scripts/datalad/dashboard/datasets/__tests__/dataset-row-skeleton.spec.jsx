import React from 'react'
import { shallow } from 'enzyme'
import DatasetRowSkeleton from '../dataset-row-skeleton.jsx'

describe('dashboard/datasets/DatasetRowSkeleton', () => {
  it('renders successfully', () => {
    expect(shallow(<DatasetRowSkeleton />)).toMatchSnapshot()
  })
})
