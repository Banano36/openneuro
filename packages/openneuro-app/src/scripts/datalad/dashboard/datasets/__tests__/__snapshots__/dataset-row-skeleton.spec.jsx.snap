// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dashboard/datasets/DatasetRowSkeleton renders successfully 1`] = `
<div
  className="panel panel-default"
>
  <Styled(div)
    className="panel-heading"
  >
    Â 
  </Styled(div)>
</div>
`;
